include/zephyr/kernel.h

```diff
@@ -4785,7 +4785,7 @@ __syscall int k_msgq_alloc_init(struct k_msgq *msgq, size_t msg_size,
 int k_msgq_cleanup(struct k_msgq *msgq);
 
 /**
- * @brief Send a message to a message queue.
+ * @brief Send a message to the end of a message queue.
```

Comment:
The commit message still has a referent to k_msgq_put_urgent instead of k_msgq_put_front.